src/Apps/Order/Routes/Shipping2/FulfillmentDetailsForm.tsx

```diff
@@ -0,0 +1,608 @@
+import * as Yup from "yup"
+import {
+  usePrevious,
+  RadioGroup,
+  BorderedRadio,
+  Spacer,
+  GridColumns,
+  Column,
+  Input,
+  AutocompleteInput,
+  Checkbox,
+  Text,
+} from "@artsy/palette"
+import {
+  AddressVerifiedBy,
+  AddressVerificationFlowQueryRenderer,
+} from "Apps/Order/Components/AddressVerificationFlow"
+import {
+  FulfillmentValues,
+  ShipValues,
+} from "Apps/Order/Routes/Shipping2/FulfillmentDetails"
+import { useShippingContext } from "Apps/Order/Routes/Shipping2/Hooks/useShippingContext"
+import { SavedAddressesFragmentContainer } from "Apps/Order/Routes/Shipping2/SavedAddresses2"
+import {
+  FulfillmentType,
+  ShippingAddressFormValues,
+} from "Apps/Order/Routes/Shipping2/Utils/shippingUtils"
+import {
+  useAddressAutocomplete,
+  AddressAutocompleteSuggestion,
+} from "Components/Address/useAddressAutocomplete"
+import { CountrySelect } from "Components/CountrySelect"
+import { RouterLink } from "System/Router/RouterLink"
+import { extractNodes } from "Utils/extractNodes"
+import {
+  useFormikContext,
+  Form,
+  FormikTouched,
+  FormikErrors,
+  Formik,
+} from "formik"
+import { compact, pick } from "lodash"
+import { useEffect, useCallback } from "react"
+import { ADDRESS_VALIDATION_SHAPE } from "Apps/Order/Utils/shippingUtils"
+import { Collapse } from "Apps/Order/Components/Collapse"
+import { FulfillmentDetailsForm_me$data } from "__generated__/FulfillmentDetailsForm_me.graphql"
+
+export interface FulfillmentDetailsFormProps {
+  // TODO: ideally we don't need to thread shipping2_me through here but that requires
+  // adding savedAdderesses to the context.
+  me: FulfillmentDetailsForm_me$data
+  initialValues: FulfillmentValues
+  verifyAddressNow: boolean
+  onAddressVerificationComplete: () => void
+  onSubmit: (values: FulfillmentValues, helpers: any) => void
+  availableFulfillmentTypes: FulfillmentType[]
+}
+
+export const FulfillmentDetailsForm = (props: FulfillmentDetailsFormProps) => {
+  return (
+    <Formik<FulfillmentValues>
+      initialValues={props.initialValues}
+      onSubmit={props.onSubmit}
+      validationSchema={VALIDATION_SCHEMA}
+    >
+      <FulfillmentDetailsFormLayout
+        me={props.me}
+        verifyAddressNow={props.verifyAddressNow}
+        onAddressVerificationComplete={props.onAddressVerificationComplete}
+        availableFulfillmentTypes={props.availableFulfillmentTypes}
+      />
+    </Formik>
+  )
+}
```

Comment:
We can now test the form from this point, which will allow moving many tests from Shipping2.jest.tsx into here. The nesting of the components is slightly changed, resulting in a large diff, but the functionality remains the same with the exception of labeling state/postal code fields.